### PR TITLE
Changes in lesson section and supplement material

### PR DESCRIPTION
**Pull request by tyagipr**
@alodahl  I added precise description of slides in the lesson section and put the video links in supplement material sections

### CODE REVIEW

File: professionalism/professionalism.md [hunk 1]
```diff
@@ -39,16 +39,51 @@ Techtonica is a simulated work environment representative of common tech industr
...
+          - No Profanity / insensitive language/images on clothes
+          - Pants should neither be tight not baggy and should not be ripped, regardless if they are in fashioin
+          - Clothes should be free from stains and wrinkles and having matching socks
+          - Workout clothes and pajamas are not allowed
+          - Guidelines for women
+              - Shirts can be sleeveless, but it should be fully cover bras and end below your pants' waistsband
+              - Shorts should be as long as your fingertip when you stand with your arm straight down, Skirts shouldn't be shorter than few inches above your knees
+              
+     - Your Arrival at Workplace
+          - If you have **planned** absence or late arrivale, notify those colleagues who will be affected with your presence as soon as you aware your absence or late arrival
```
Comment (manaswinidas) [inline professionalism/professionalism.md]: Replace 'arrivale' with 'arrival'

File: professionalism/professionalism.md [hunk 2]
```diff
@@ -39,16 +39,51 @@ Techtonica is a simulated work environment representative of common tech industr
...
+          - Arrive early enough so that you can give yourself sometime to get settled before the start of a workday or meeting
+          
+     - Your behaviour during the Workday
+          - Do your best, try to avoid "stealing time" , which means doing non-work things during the non-break hours in the workplace
+          - Don't Complain much while you stuck in something, try to learn from your colleagues and expertise how to find out solutions of a particular problem
```
Comment (manaswinidas) [inline professionalism/professionalism.md]: 'while you are stuck in something' and 'experts' in place of 'expertise'

File: professionalism/professionalism.md [hunk 3]
```diff
@@ -39,16 +39,51 @@ Techtonica is a simulated work environment representative of common tech industr
...
+          - You should treat everyone respectfully even if you don't even like them
+          - Make sure you submit your assignment and given task timely, ensure yourself that you have enough skills and tools for the assigned task or work, if you don't have just feel free to ask your colleagues and take their help
+          - If you can’t make a deadline, let those affected know as soon as you realize it
+       
+     - Your Departure from Workplace
+          - Normally working hours is 8 ,and because you spent one hour in break, So try to compensate it with working 9 hours
+          - Try to leave after you finishes you assigned day task, if you can't complete it then speak to your manager about re-setting the expectations
```
Comment (manaswinidas) [inline professionalism/professionalism.md]: 'after you finish your assigned task'

**Comment by manaswinidas** [comment]
Please mention the issue number so that it becomes easier for us to add scores

**Comment by tyagipr** [comment]
Done! @manaswinidas #617 
I will remove the duplicate one

File: professionalism/professionalism.md [hunk 4]
```diff
@@ -67,19 +67,19 @@ Techtonica is a simulated work environment representative of common tech industr
               - Shorts should be as long as your fingertip when you stand with your arm straight down, Skirts shouldn't be shorter than few inches above your knees
               
      - Your Arrival at Workplace
-          - If you have **planned** absence or late arrivale, notify those colleagues who will be affected with your presence as soon as you aware your absence or late arrival
+          - If you have **planned** absence or late arrival, notify those colleagues who will be affected with your presence as soon as you aware your absence or late arrival
           - Arrive early enough so that you can give yourself sometime to get settled before the start of a workday or meeting
           
      - Your behaviour during the Workday
           - Do your best, try to avoid "stealing time" , which means doing non-work things during the non-break hours in the workplace
-          - Don't Complain much while you stuck in something, try to learn from your colleagues and expertise how to find out solutions of a particular problem
```
Comment (manaswinidas) [inline professionalism/professionalism.md]: 'while you *are* stuck in something or you're stuck in something'

**Comment by tyagipr** [comment]
Done!

**Pull request merged**